ui/pages/onboarding-flow/privacy-settings/privacy-settings.js

```diff
@@ -138,8 +138,10 @@ export default function PrivacySettings() {
     setUseTransactionSimulations(isTransactionSimulationsEnabled);
     dispatch(setPetnamesEnabled(turnOnPetnames));
 
-    if (!isProfileSyncingEnabled && participateInMetaMetrics) {
-      dispatch(performSignIn());
+    if (externalServicesOnboardingToggleState) {
+      if (!isProfileSyncingEnabled && participateInMetaMetrics) {
+        dispatch(performSignIn());
```

Comment:
Hotfixes to ensure we do not all these methods.

A separate PR/s will be out soon that will handle:
1. Better UI states (our toggles will either need to be disabled or not visible if the External Services toggle is not enabled)
2. Improved Logic and Controllers - ensure at a controller level that our MM services are not called when this switch is disabled.